docs/HowTo/Monitoring.md

```diff
@@ -0,0 +1,84 @@
+---
+description: Monitor Rollups
+---
+
+# Monitor ConsenSys Rollups
+
+You can use ConsenSys Rollups with [Prometheus](https://prometheus.io/) time-series database to record API usage metrics.
+You can visualize the recorded data using an existing dashboard tool such as [Grafana](https://grafana.com/).
+
+## Record API metrics
+
+ConsenSys Rollups can record the following usage metrics for each request routed through its Handlers:
```

Comment:
To be precise this metric shows how much time it took to decode request and store it mempool. 
Actual account creation or money order operation can take longer, because it depends on how soon batch will be filled, processed and submitted to blockchain.

But this is relevant for engine, probably manager counts whole time it took for request to be finalized. @jpnovais @Filter94  could you please confirm it?